tendrl_gluster_peers_bricks.yml

```diff
@@ -1,9 +1,10 @@
 ---
 #
-# * Install Tendrl server and tendrl-gluster nodes.
-# * Configure gluster trusted storage pool and create bricks 
-#   on all available devices on gluster nodes.
+# Install Tendrl server and tendrl-gluster nodes.
+# Configure gluster trusted storage pool and create bricks
+# on all available devices on gluster nodes.
 #
 - include: qe_pre_installation_tasks.yml
-- include: tendrl_gluster.yml
```

Comment:
I would rather have a single playbook, clearly maintained in git repo, for each jenkins job. So I'm in favor of keeping `tendrl_gluster.yml` playbook.